ajax/libs/timbre/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "timbre",
+  "version": "14.11.25",
+  "description": "JavaScript library for objective sound programming",
+  "author": "nao yonamine <mohayonao@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mohayonao/timbre.js.git"
+  },
+  "filename": "timbre.node.min.js",
+  "license": "MIT",
+  "dependencies": {
+    "readable-stream": "^1.0.33",
+    "speaker": "^0.2.1"
+  },
+  "keywords": [
+    "timbre",
+    "T-Object"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mohayonao/timbre.js.git",
+    "basePath": "",
+    "files": [
+      "timbre.node*.js"
+    ]
+  },
+  "requiredFiles": [
```

Comment:
You have point the minified file in "filename" field, no need to add "requiredFiles".